src/proto/uprotocol/upayload.rs

```diff
@@ -11,13 +11,100 @@
  * SPDX-License-Identifier: Apache-2.0
  ********************************************************************************/
 
+use mediatype::{names::*, MediaType, Name};
 use prost::Message;
 use prost_types::Any;
 
-use crate::{
-    cloudevent::serializer::SerializationError,
-    uprotocol::{Data, UPayload, UPayloadFormat},
-};
+pub use crate::types::parsingerror::ParsingError;
+pub use crate::types::serializationerror::SerializationError;
+use crate::uprotocol::{Data, UPayload, UPayloadFormat};
+
+const SUBTYPE_PROTOBUF: Name = Name::new_unchecked("protobuf");
```

Comment:
In theory, that would be the way to go. However, *prost-build* does not support custom options (yet). For the time being, we can either keep the const definitions in place or migrate the whole SDK to using the *rust-protobuf* crate instead of *prost-build*. *rust-protobuf* seems to have rudimentary support for extensions/custom options ...

@AnotherDaniel I had already proposed to migrate to *rust-protobuf* several weeks before ...
WDYT?

Reply:
As the rust-protobuf project is not dead after all, some recent commits coming in (it looked dead when I checked it out last summer), I'm good with switching to using that. Do you still have the code from that experiment lying around?